lib/takeaway.rb

```diff
@@ -0,0 +1,38 @@
+require_relative './menu.rb'
+
+class Takeaway
+  attr_reader :basket
+
+  def initialize
+    @menu = Menu.new
```

Comment:
For this you can initialise with and argument to set the menu. Then this class will no longer `require` "menu.rb"